src/View/Helper/PaginatorHelper.php

```diff
@@ -0,0 +1,96 @@
+<?php
+namespace BootstrapUI\View\Helper;
+
+use Cake\View\View;
+
+class PaginatorHelper extends \Cake\View\Helper\PaginatorHelper
+{
+
+    /**
+     * Constructor. Overridden to merge passed args with URL options.
+     *
+     * @param \Cake\View\View $View The View this helper is being attached to.
+     * @param array $config Configuration settings for the helper.
+     */
+    public function __construct(View $View, array $config = [])
+    {
+        $this->_defaultConfig['templates'] = [
+            'nextActive' => '<li class="next"><a rel="next" aria-label="Next" href="{{url}}">' .
+                            '<span aria-hidden="true">{{text}}</span></a></li>',
+            'nextDisabled' => '<li class="next disabled"><a href=""><span aria-hidden="true">{{text}}</span></a></li>',
+            'prevActive' => '<li class="prev"><a rel="prev" aria-label="Previous" href="{{url}}">' .
+                            '<span aria-hidden="true">{{text}}</span></a></li>',
+            'prevDisabled' => '<li class="prev disabled"><a href=""><span aria-hidden="true">{{text}}</span></a></li>',
+            'current' => '<li class="active">{{text}} <span class="sr-only">(current)</span></li>',
```

Comment:
This actually needs to be `<li class="active"><span>{{text}} <span class="sr-only">(current)</span></span></li>`  http://getbootstrap.com/components/#disabled-and-active-states

[insert obligatory comment about BS and html tag soup]